### PR TITLE
ApiDoc

### PR DESCRIPTION
**Pull request by kanghua-wangchanghao**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,4 @@
 !var/SymfonyRequirements.php
 /vendor/
 /web/bundles/
+.idea/
```
Comment (yceruto) [inline .gitignore]: This should be excluded in you `.git/info/exclude` or global

File: app/config/config.yml [hunk 2]
```diff
@@ -89,3 +89,6 @@ swiftmailer:
     username:  "%mailer_user%"
     password:  "%mailer_password%"
     spool:     { type: memory }
+
+#开启ApiDoc配置
+nelmio_api_doc: ~
```
Comment (yceruto) [inline app/config/config.yml]: Missing newline at end of file

File: app/config/routing.yml [hunk 3]
```diff
@@ -25,3 +25,8 @@ homepage:
         _controller: FrameworkBundle:Template:template
         template:    default/homepage.html.twig
         _locale:     '%locale%'
+
+#配置ApiDoc路由
```
Comment (yceruto) [inline app/config/routing.yml]: The comments should be in English

File: src/AppBundle/Controller/BlogController.php [hunk 4]
```diff
@@ -20,6 +20,7 @@
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Nelmio\ApiDocBundle\Annotation\ApiDoc;//添加ApiDoc依赖
```
Comment (yceruto) [inline src/AppBundle/Controller/BlogController.php]: Unnecessary comments

File: web/app.php [hunk 5]
```diff
@@ -28,7 +28,7 @@
 // $loader->unregister();
 // $apcLoader->register(true);
 
-$kernel = new AppKernel('prod', false);
+$kernel = new AppKernel('prod', true);//因为缓存无法启动ApiDoc
```
Comment (yceruto) [inline web/app.php]: Should be reverted (no make sense), in prod env `debug` should be `false`.

**Comment by javiereguiluz** [comment]
@kanghua-wangchanghao we appreciate your contribution, but sadly we can't accept this contribution. The main reason is that this bundle is used to generate the documentation of the API ... but we don't have an API in this application. Besides, for this kind of important changes, we prefer to create an Issue first so we can discuss about it and once approved, then submit the pull request.

So, I must close this pull request, but I hope you understand it. Thanks!

**Pull request closed without merging**